app/routes/pizza.router.ts

```diff
@@ -0,0 +1,31 @@
+import {Router} from 'express';
+import initializeDb from "../config/db";
+import * as Pizza from '../controller/pizza';
+import {AuthVerify, handleUnauthorizedError} from "../middleware/auth";
+import {Middleware} from "Diwala";
+
+
+const pizzaRouter = Router();
+
+const handleUnhandledError: Middleware = (
+    err: any,
+    _req: any,
+    res: any,
+    _next: any
+  ) => {
+    console.error("unhandled error", err);
+    return res.status(500).end();
+  };
+
+initializeDb();
+pizzaRouter.post('/pizza', AuthVerify, Pizza.createPizza)
+pizzaRouter.get('/get-pizza', Pizza.getPizzas)
+pizzaRouter.get('/get-pizza-by-id/:id', Pizza.getPizzabyId)
```

Comment:
I suggest this, `pizzaRouter.get('/pizza/:id', Pizza.getPizzaById)` That works fine because the parametere differentiates things